src/editor/index.js

```diff
@@ -18,6 +18,10 @@ For default config and extensions (and available options) available to
 `setConfig()`, see the file `docs/tutorials/ConfigOptions.md`
 */
 
+// eslint-disable-next-line node/file-extension-in-import
+import 'core-js/stable';
+// eslint-disable-next-line node/file-extension-in-import
+import 'regenerator-runtime/runtime';
```

Comment:
This style of import suffers from causing one not to be able to have working browser source since browsers only recognize relative paths (import maps can help with this, but it would be cumbersome currently: https://github.com/WICG/import-maps/issues/126 ). One has to go through the build step (or have a test server which processes these).

You could use `./node_modules` paths however, and the potential issue of non-flattened node_modules should be less likely given that svgedit is less likely to be a transitive dependency.

Reply:
See comment above. This branch is not yet GTM.

Reply:
changed the web server so src is usable (including for tests)

Reply:
Ok, but note that Netlify appears per https://community.netlify.com/t/support-guide-can-i-run-a-web-server-http-listener-and-or-database-at-netlify/3078 to be only for static files. So our demos wouldn't be able to work with the ESM editor (unless the editor used the ESM distribution files, and with source maps back to source so people reporting bugs could at least help us identify where the problems lay).

Reply:
> Ok, but note that Netlify appears per https://community.netlify.com/t/support-guide-can-i-run-a-web-server-http-listener-and-or-database-at-netlify/3078 to be only for static files. So our demos wouldn't be able to work with the ESM editor (unless the editor used the ESM distribution files, and with source maps back to source so people reporting bugs could at least help us identify where the problems lay).

The demo hosted by Netlify will work from any of the bundle in the dist folder.